src/Ledger/PParams.lagda

```diff
@@ -450,7 +436,7 @@ something) and if it preserves well-formedness.
 
 \begin{figure*}[ht]
 \begin{AgdaMultiCode}
-\begin{code}[hide]
```

Comment:
This was hiding both the record name and the field

Reply:
Yes, but I thought it would be a good idea to reveal all record names whose fields are shown... but maybe not.  Anyway, I'll hide this one again.

Reply:
Makes sense!